src/compounds/RequestItem/RequestItem.jsx

```diff
@@ -4,26 +4,32 @@ import Badge from '../../components/Badge/Badge'
 import Image from '../../components/Image/Image'
 import TextBox from '../../components/TextBox/TextBox'
 import Title from '../../components/Title/Title'
-import './request-item.css'
+import './request-item.scss'
 
-const RequestItem = React.forwardRef(({ createdAt, description, href, img, title, status, updatedAt }, ref) => {
+const RequestItem = React.forwardRef(({ createdAt, description, href, img, index, title, status, updatedAt }, ref) => {
   const { backgroundColor, text, textColor } = status
   const image = { ...img, height: 70, width: 70 }
 
   return (
-    <article className='request-item mt-2'>
-      <Image {...image} />
-      <div className='request-item-details'>
-        <a href={href} ref={ref} className='link pointer-cursor'>
-          <Title title={title} size='small' />
-        </a>
-        <TextBox text={description} />
+    <article
+      className={`request-item p-3 d-flex flex-column flex-md-row justify-content-between gap-3 bg-light 
+      ${index % 2 === 0 ? ' bg-light' : ' bg-light-2'}`}
```

Comment:
since we're only passing `index` to apply a style, can we instead try using the `:nth-child()` selector in the scss file to achieve this result?

Reply:
i tried that, and it looks great in the component library but i was just trying to think ahead for changing these colors per instance. 

the variable for light-2 and all associated utility classes has been added to the webstore. its value depends on the value for "light," so whatever the user sets "light" to in their webstore theme, light-2 will be slightly darker and these rows will remain 2 slightly different shades of the same color, which is the intention.

if the user wants them to be 2 totally different colors they can do that too, they would just need to override both the light & light-2 variables in the webstore with their chosen colors

Reply:
can we not use the utility class variables in the scss file?

Reply:
no, because that requires importing the bootstrap-preview.js file into the `request-item.scss` file to have access to those variables in the component library. we do not want the actual values from the component library to apply to the webstore or any of the bootstrap imports to be imported twice. Thats why we use the utility classes, since these are also generated in the webstore, the class in the webstore will take precedence

Reply:
boooo to bootstrap for that. ok.